### ISSUE TITLE
Translator's copyright in translation as derivative work

### ISSUE THREAD

**Issue by dentarthur** [DavidMc1948/TPM#4]
Please track down translator Stephen Trask eg from Library of Congress authority records. Will need date of death for trade publisher to confirm in public domain (50 or 70 years after death or whatever the relevant copyright absurdity in Germany, and UK and publisher location says for translators). Borchardt died more than 85 years ago (1868-1932) but translator could have been younger and lived longer.

https://en.m.wikipedia.org/wiki/Julian_Borchardt 

Goal is a scholarly trade publisher (perhaps together with Engels synopsis, included in same index). That will make a major difference to awareness of both the work itself and of Maksakovsky Institute so needs priority over mediawiki and marxists.org despite resulting delay. Any value added we (mainly you) can do could make a real difference for the trade publisher.

I will not be starting to write introduction till later as more urgent to reduce other delays. Should not take long once those are taken care of.

Please phone to discuss this and other recent issues.

**Comment by DavidMc1948** [comment]
Can't find him at http://authorities.loc.gov.

**Comment by DavidMc1948** [comment]
No luck finding Trask.

**Comment by DavidMc1948** [comment]
From forgettenbooks.com FAQ for what its worth.

"We check each book to ensure it is not within copyright before creating a derivative edition. However, in rare cases a book may be incorrectly marked as out-of-copyright. If this has happened and you are the copyright holder then please contact us using the form below."

https://www.forgottenbooks.com/en/help

**Comment by dentarthur** [comment]
Thanks. Closing thread as LC authorities best place to check and presumably nobody has complained to Forgottenbooks or internet archive.

BTW I looked at their copy of TPM from "read" online and downloaded 296pp, 6.9MB pdf which has ludicrous copyright notice for their "derivative work" of simply reproducing scan at internet archive. Includes most of the pages, interrupted by ads urging payment of about USD $9 per month for access to about 800,000 books - presumably Gutenberg and internet archive. I gather there are quite a few of these print on demand operations and they have a standard api to actual print shops and use the apis provided by internet archive and Gutenberg.

Will definately need substantial value adding to get a trade publisher since already available free.
